enterprise-ws-spec/src/main/asciidoc/Objectives.adoc

```diff
@@ -4,89 +4,81 @@ This section lists the high level objectives of this specification.
-interoperate using the transport standards declared within WSDL and the
-JAX-RPC or JAX-WS runtime with Web Services for Java EE applications.
 
 === Service Development Goals
 
 The service development model defines how web service implementations
-are to be developed and deployed into existing Java EE containers and
+are to be developed and deployed into existing Jakarta EE containers and
 includes the following specific goals:
 
 * How the Web service has been implemented should be transparent to the
 Web service client. A client should not have to know if the Web service
-has been deployed in a Java EE or non-Java EE environment.
-* Because the Web service implementation must be deployed in a Java EE
+has been deployed in a Jakarta EE or non-Jakarta EE environment.
+* Because the Web service implementation must be deployed in a Jakarta EE
 container, the class implementing the service must conform to some
 defined requirements to ensure that it does not compromise the integrity
 of the application server.
-* JAX-RPC defines three server side run time categories, J2SE based
-JAX-RPC Runtime, Servlet Container Based JAX-RPC Runtime, and Java EE
-Container Based JAX-RPC Runtime. This specification defines the Java EE
-container based (Web and EJB) runtime such that it is consistent with
-the Servlet Container based model defined by the JAX-RPC specification.
-* This specification is amended to define the Java EE container based
-(Web and EJB) runtime such that it is consistent with the programming
-model defined by the JAX-WS specification.
+* This specification defines the Jakarta EE container based (Web and EJB)
+runtime such that it is consistent with the programming model defined by
+the Jakarta XML Web Services specification.
 * Support mapping and dispatching SOAP 1.1 or 1.2 requests to methods on
-Java EE Stateless or Singleton(only JAX-WS) Session Beans.
+Jakarta EE Stateless or Singleton Session Beans.
 * Support mapping and dispatching SOAP 1.1 or 1.2 requests to methods on
-JAX-RPC or JAX-WS Service Endpoint classes in the Web Container.
+Jakarta XML Web Services Service Endpoint classes in the Web Container.
 
 === Service Deployment Goals
 
 * Web service deployment is declarative. We do this through extending
-the Java EE model for deployment descriptors and EAR file format. These
+the Jakarta EE model for deployment descriptors and EAR file format. These
 changes are minimized, however.
-* Web service deployment is supported on Java EE environments.
+* Web service deployment is supported on Jakarta EE environments.
 * Deployment requires that a service be representable by WSDL.
 Deployment requires a WSDL file. The deployment of Web services must
 support:
 
 * those who wish to deploy a Web service as the focus of the deployment
-* those who wish to expose existing, deployed Java EE components as a
+* those who wish to expose existing, deployed Jakarta EE components as a
```

Comment:
these last two items (starting with 'those') are wrongly formatted they're supposed to be nested under/related only to the "The deployment of Web services must support:" sentence; they should not be at the same level as previous items